File: Golang/2535-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.go
func differenceOfSum(nums []int) int {
    elementSum, digitSum := 0, 0
    for _, num := range nums {
        elementSum += num
        for num > 0 {
            digitSum += num % 10
            num /= 10
        }
    }
    if elementSum - digitSum < 0 {
        return -1 * (elementSum - digitSum)
    }
    return (elementSum - digitSum)
}

File: Python/2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        element_sum = digit_sum = 0
        for num in nums:
            element_sum += num
            while num > 0:
                digit_sum += (num % 10)
                num = num // 10
        if element_sum > digit_sum:
            return element_sum - digit_sum
        else:
            return digit_sum - element_sum